src/coscale/api/server.go

```diff
@@ -83,6 +84,52 @@ func (api *Api) UpdateServer(server *Server) (string, error) {
 	return api.GetObject("server", server.ID)
 }
 
+// GetServerGroupByPath returns a server group by the hierarchy.
+func (api *Api) GetServerGroupByPath(path string) (string, error) {
+
+	var serverGroup *ServerGroup
+
+	groupNames := strings.Split(path, "/")
+	for i, groupName := range groupNames {
+
+		var query string
+
+		if i == 0 {
+			query = "selectByRoot=true"
+		} else if serverGroup != nil && serverGroup.ID > 0 {
+			query = fmt.Sprintf("selectByParent_id=%d", serverGroup.ID)
+		} else {
+			return "", fmt.Errorf("Failed to get the parent for ServerGroup: %s", groupName)
+		}
+
+		api.SetQueryString(query)
+		result, err := api.GetObjectByName("servergroup", groupName)
+		if err != nil {
+			return "", err
+		}
+		// For the last group just return it.
+		if i == len(groupNames)-1 {
+			return result, nil
+		}
+
+		tmp := make([]*ServerGroup, 1)
+		err = api.HandleResponse([]byte(result), false, &tmp)
+		if err != nil {
+			return "", err
+		}
+		if len(tmp) != 1 {
+			return "", fmt.Errorf("Failed to get the parent for ServerGroup: %s", groupName)
+		}
+		serverGroup = tmp[0]
+	}
+
+	if serverGroup != nil {
+		return "", fmt.Errorf("Failed to get the child of group: %s", serverGroup.Name)
+	}
+
+	return "", fmt.Errorf("Failed to get the group: %s", path)
```

Comment:
When you do `coscale-cli servergroup get --name "Test"` without a server group named Test, it returns

```[]```

So I think we should do the same here. If the 1 of the server groups in the tree is not found, we should return 

```[]```